core/src/test/java/io/questdb/test/cairo/mv/MatViewTest.java

```diff
@@ -6767,6 +6768,14 @@ private static void assertCannotModifyMatView(String updateSql) {
         }
     }
 
+    private static void assertSqlError(String badSql, String expectedErrorMessage) {
```

Comment:
constant args, it is either unnecessarily generic method or insufficient testing